src/context/schemas/transactionresult.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/transactionresult.schema.json",
+  "type": "object",
+  "title": "TransactionResult",
+  "allOf": [{ "$ref": "context.schema.json#" }],
+  "properties": {
+    "type": { "const": "fdc3.transactionResult" },
+    "status": {
+      "anyOf": [
+        {
+          "type": "string",
+          "enum": ["Created", "Deleted", "Updated"]
```

Comment:
I think this needs to include error statuses - unless we decide that the raised intent itself should return an error.

I think we should aim to define the statuses, rather than allow any string (as the anyOf is currently allowing).

I would also suggest adding a string message field - particularly for error statuses.

Reply:
Hi Chris. Thanks for the feedback 

I see where you're coming from but I'm not sure how best to update the json. Would this do the trick?

```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "$id": "https://fdc3.finos.org/schemas/next/transactionresult.schema.json",
  "type": "object",
  "title": "TransactionResult",
  "allOf": [{ "$ref": "context.schema.json#" }],
  "properties": {
    "type": { "const": "fdc3.transactionResult" },
    "status": {
          "type": "string",
          "enum": ["Created", "Deleted", "Updated", "Failed"]
        },
    "context": { "$ref": "context.schema.json#" }
  },
    "message": {
      "type": "string"
    },
  "required": ["status", "context"]
}
```

Reply:
Yes, LGTM, although `context` _could_ be optional I suppose... I guess the question is: do we want to force something to be returned?  

I'm also wondering about the name of that field. Perhaps it should more strongly imply a particular use (e.g. `result` or `output`). Just a thought at this stage.

Reply:
I think we probably do want something to come back. If they really don't want to return something I suppose they can put [`fdc3.nothing`](https://fdc3.finos.org/docs/next/context/ref/Nothing) in ;-)